doc/classes/Physics2DServer.xml

```diff
@@ -216,7 +216,7 @@
 			<argument index="2" name="disable" type="bool">
 			</argument>
 			<description>
-				Disables a given shape in this area if [code]disable is true[/code]
+				Disables a given shape in this area if [code]disable[/code] is [code]true[/code].
```

Comment:
This feels redundant to me. Is it really needed to explicitly explain the obviously named boolean? Also, the area is an argument, so I wouldn't say "this area".

Disables a given shape in an area.